src/data_access/FirebaseSettingsDataAccessObject.java

```diff
@@ -0,0 +1,11 @@
+package data_access;
+
+import entity.User;
+import use_case.settings.SettingsDataAccessInterface;
+
+public class FirebaseSettingsDataAccessObject implements SettingsDataAccessInterface {
+  @Override
+  public String op(User user) {
```

Comment:
This comment is just here to acknowledge that this API is not the final design and will be removed